zubhub_backend/zubhub/projects/models.py

```diff
@@ -249,3 +249,27 @@ def save(self, *args, **kwargs):
             uid = uid[0:floor(len(uid) / 6)]
             self.slug = slugify(self.title) + "-" + uid
         super().save(*args, **kwargs)
+
+
+class UnpublishmentReason(models.Model):
+    id = models.UUIDField(primary_key=True,
+                          default=uuid.uuid4,
+                          editable=False,
+                          unique=True)
+    description = models.CharField(max_length=250)
+
+
+class Unpublishment(models.Model):
+    id = models.UUIDField(primary_key=True,
+                          default=uuid.uuid4,
+                          editable=False,
+                          unique=True)
+    reason = models.ForeignKey(UnpublishmentReason,
```

Comment:
Since the form on the frontend allows for multiple reasons, I believe this should be a ManyToManyField.